File: tests/dynamic_fixtures/js/xss_unsupported.js
// XSS — unsupported fixture.
// Entry is a class method rather than a top-level function.
// Test sets confidence = Low to get Unsupported(ConfidenceTooLow).
// Entry: TemplateEngine.render(input)  Cap: HTML_ESCAPE
// Expected verdict: Unsupported

class TemplateEngine {
    render(input) {
        return '<html><body>' + input + '</body></html>';
    }
}

module.exports = { TemplateEngine };

File: tests/dynamic_fixtures/go/xss_unsupported.go
// XSS — unsupported fixture.
// Entry is a method on a struct; test sets confidence = Low.
// Expected verdict: Unsupported

package entry

import "fmt"

type Renderer struct{}

func (r *Renderer) Render(input string) {
	fmt.Print("<html><body>" + input + "</body></html>\n")
}
